test/CMakeLists.txt

```diff
@@ -79,5 +79,4 @@ macro (add_subdirectories)
         endif ()
     endforeach ()
 endmacro ()
-
```

Comment:
But it's not even the end of the file... We can look at it together at the SeqAn meeting!